notifications/06-13-20-0900.md

```diff
@@ -0,0 +1,26 @@
+**What is happening?**
+
+RFC 194163 
+
+CITZ - Standard Change #33 MCS PROD - MCS OS Patching to PROD cluster and NFS server
+
+**When?**
+
+Starting at Saturday June 13th @ 0800 and finishing at Friday June 19th @ 1600.
+
+**Will there be an impact on the Platform apps?**
+
+Some infrastructure services like metrics and Kibana will be briefly unavailable or have reduced performance during some operations. The app nodes will be drained in sequence during the weekdays and pods may experience some brief outages as pods are moved between nodes.
+
+**Do I need to do anything?**
+
+Monitor the health of your apps during the change. Refer to [Post-Outage Application Checklist](https://developer.gov.bc.ca/Post-Outage-Application-Health-Checklist) for health check tips.
+
+**Check the #devops-alert channel for the announcement of when the change is complete and check the health of your app.**
+
+**Where do I get help if my app doesn't work after the change is complete?**
+
+Each Platform application has an assigned DevOps Specialist within the Ministry so contact them first. If you don't know who your assigned DevOps Specialist, check with the app's Product Owner.
+
+The DevOps Specialist will troubleshoot the issue with the app and if they need help, they will reach out to the Platform Services Team and the Developer Community in Rocket.Chat as per these [RocketChat Channel Use Guidelines](
+https://developer.gov.bc.ca/Getting-human-support-for-issues-not-covered-by-devops-requests).
```

Comment:
If you have any question or concerns about this maintenance, please let the Platform Services Team know in the #devops-operations channel in RocketChat or contact Olena Mitovska, Product Owner for Platform Services, directly.